src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationLineWrappedRecordDeclaration.java

```diff
@@ -32,7 +32,7 @@ private static record ConcreteRecord2(
         String greeting                                                             //indent:8 exp:8
 ) implements Interf {                                                          //indent:0 exp:4 warn
         @Override                                                                   //indent:8 exp:8
-    public String sayHello() {                                                      //indent:4 exp:4
+        public String sayHello() {                                                  //indent:8 exp:8
```

Comment:
Don'e change the previous signature, revert the code, and just update the warn comments if you are getting such.

Reply:
Done